js/samples/quickstart-csharp/Controllers/HomeController.cs

```diff
@@ -2,6 +2,7 @@
 using Microsoft.AspNetCore.Mvc;
 using QuickstartSampleWebApp.Models;
 using Microsoft.Identity.Client;
+using System.Text;
```

Comment:
usually we keep all System using statements above other statements.

Reply:
makes sense, done